dashboard/src/components/Catalog/CatalogItem.test.tsx

```diff
@@ -34,7 +35,7 @@ it("should render an item", () => {
 });
 
 it("should use the default placeholder for the icon if it doesn't exist", () => {
-  const chartWithoutIcon = { ...defaultChart };
```

Comment:
oh, so that does not work then?

Reply:
nope, and I just realized that `Object.assign` doesn't work either 😞. The problem is that it mutates inner objects:

```
> a = { a: { b: 1 }}
{ a: { b: 1 } }
> b = {...a}
{ a: { b: 1 } }
> b.a.b = 2
2
> a
{ a: { b: 2 } }
```

The only thing that I found that behaves as expected is lodash `cloneDeep`:

```
> b = u.cloneDeep(a)
{ a: { b: 1 } }
> b.a.b = 2
2
> a
{ a: { b: 1 } }
```